Fuenf freie Seiten bei starkem NFS-Traffic ist fuer ISDN viel zu wenig.

Notwendig, wenn der ISDN-Server als NFS-Client arbeitet.

--- /pub/src/linux/kernel/linux-1.1/mm/swap.c	Fri Jan 20 00:12:16 1995
+++ mm/swap.c	Sat Jan 21 08:35:01 1995
@@ -646,8 +652,9 @@
 			priority = GFP_ATOMIC;
 		}
 	}
-	reserved_pages = 5;
-	if (priority != GFP_NFS)
+	if (priority == GFP_NFS)
+		reserved_pages = min_free_pages>>1;
+	else
 		reserved_pages = min_free_pages;
 	save_flags(flags);
 repeat:
